### PR TITLE
[Doc] Add Chinese version of 1_exist_data_model doc

### PR DESCRIPTION
**Pull request by AronLin**
Add Chinese version of 1_exist_data_model doc

### CODE REVIEW

File: docs_zh-CN/1_exist_data_model.md
Comment (ZwwWayne) [inline docs_zh-CN/1_exist_data_model.md:115]: 这是使用摄像头实时图片的推理脚本

File: docs_zh-CN/1_exist_data_model.md
Comment (ZwwWayne) [inline docs_zh-CN/1_exist_data_model.md:135]: This file does not strictly follow the markdown format

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:45]: delete 于

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:47]: Deleted, batch is currently supported.

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:66]: translation

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:52]: You may want to add some descriptions, because asynchronous io has been integrated into image_demo.py.

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:160]: Space

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:216]: ?

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:436]: #id9 ?

File: docs_zh-CN/1_exist_data_model.md
Comment (ZwwWayne) [inline docs_zh-CN/1_exist_data_model.md:337]: 支持在xxxx的情况下

File: docs_zh-CN/1_exist_data_model.md
Comment (ZwwWayne) [inline docs_zh-CN/1_exist_data_model.md:373]: 批量推理

File: docs_zh-CN/1_exist_data_model.md
Comment (ZwwWayne) [inline docs_zh-CN/1_exist_data_model.md:493]: 很好的->常见的

**Comment by ZwwWayne** [comment]
LGTM and can be merged after resolving comments, @hhaAndroid may check again before approval

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:92]: space

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:114]: space

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:125]: space

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:104]: space

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:134]: ...

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:148]: ...

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:158]: delete

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:162]: miss link

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:164]: 我们建议将数据集下载，然后解压缩到项目外部的某个文件夹内

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:248]: 不需要另起一行

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:250]: same

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:252]: 指明

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:321]: space

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:334]: space

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:432]: Link redirection failed

File: docs_zh-CN/1_exist_data_model.md
Comment (hhaAndroid) [inline docs_zh-CN/1_exist_data_model.md:451]: space

File: docs_zh-CN/1_exist_data_model.md
Comment (AronLin) [reply docs_zh-CN/1_exist_data_model.md:432]: I tried and succeed.

File: docs_zh-CN/1_exist_data_model.md
Comment (AronLin) [reply docs_zh-CN/1_exist_data_model.md:163]: Added link at line 158.

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**
